src/Symfony/Component/Lock/Bridge/DynamoDb/README.md

```diff
@@ -0,0 +1,12 @@
+Amazon DynamoDB Lock
+====================
+
+Provides Amazon DynamoDB integration for Symfony Lock.
```

Comment:
We can add a small doc for the DSN to use, as we do for some bridges ([example for Brevo](https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/Notifier/Bridge/Brevo/README.md)).